src/app/components/NumberPickers/NumberPickers.tsx

```diff
@@ -0,0 +1,14 @@
+import { getAllData } from '../../test-form/actions/form-actions';
+
+const NumberPickers = async () => {
+  const numberPicker = await getAllData();
```

Comment:
not sure why this is called 'NumberPickers' - it doesn't seem to be doing anything related to picking numbers, it just gets some form data and renders it?